pegasus/sites.v3/code.org/public/css/page.css

```diff
@@ -42,107 +42,3 @@
 }
 
 /* End of custom styling for included user_stats. */
-
-/* Duplicated from dashboard for included mini blocks, which isn't ideal. */
-
-.miniblock {
-  background-color: #586771;
-  height: 100px;
-  margin-left: 0px;
-}
-.miniblock .heading {
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
-  padding: 10px 0px 10px 0px;
-}
-.miniblock .heading .whitetext {
-  color: white;
-}
-.miniblock h3 {
-  margin: 0;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
-.miniblock .smallwhitetext {
-  color: white;
-  font-size: 13px;
-  padding-right: 10px;
-  font-family: 'Gotham 4r', sans-serif;
-}
-.miniblock .imgspan {
-  margin-left: 0px;
-}
-
-.miniblock-tall {
-  height: 150px;
-}
-
-.miniblock-short {
-  height: 170px;
-}
-
-.miniblock-short h3 {
-  margin-bottom:10px;
-}
-
-.miniblock .smalltext {
-  font-size: 13px;
-  padding-right: 10px;
-  font-family: 'Gotham 4r', sans-serif;
-}
-
-.miniblock + .miniblock {
-  margin-left: 40px;
-}
-
-.miniblock a:link,
-.miniblock a:visited,
-.miniblock a:hover,
-.miniblock a:active {
-  color: #fff;
-}
-
-/* End of duplicated from dashboard for included mini blocks. */
-
-/* Additional styling to make mini blocks work. */
-
-.span1 {  width: 60px;  }
```

Comment:
trying to think whether there's anywhere else in pegasus that might rely on this.  probably none, but worth pondering.